hydra_provider/proto/data_feed.proto

```diff
@@ -25,6 +25,11 @@ message SerialDataFeedStatus {
 service RandomDataFeed {
 	rpc start(Empty) returns(Empty);
 	rpc stop(Empty) returns(Empty);
+	rpc is_running(Empty) returns(RandomDataFeedStatus);
```

Comment:
Can you use the same get status pattern as serial instead of this for consistency? The more you repeat a pattern the easier it becomes to maintain it/migrate it to a newer pattern.

Reply:
sure why not :+1:

Reply:
Not just the name but the configuration object as well have that for random service

Reply:
done